### PR TITLE
backport: trivial backports Feb 26 2022

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
(this will likely fail ci, have only built locally)

### CODE REVIEW

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: doc/rapidcheck.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# RapidCheck property-based testing for Bitcoin Core
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply doc/rapidcheck.md:1]: dropped

File: src/qt/bitcoingui.cpp [hunk 2]
```diff
@@ -895,6 +895,10 @@ void BitcoinGUI::addWallet(WalletModel* walletModel)
 void BitcoinGUI::removeWallet(WalletModel* walletModel)
 {
     if (!walletFrame) return;
+
+    labelWalletHDStatusIcon->hide();
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/qt/bitcoingui.cpp]: dropped

File: src/Makefile.test.include [hunk 3]
```diff
@@ -356,6 +357,12 @@ test_fuzz_eval_script_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
 test_fuzz_eval_script_LDFLAGS = $(RELDFLAGS) $(AM_LDFLAGS) $(LIBTOOL_APP_LDFLAGS) $(LDFLAGS_WRAP_EXCEPTIONS)
 test_fuzz_eval_script_LDADD = $(FUZZ_SUITE_LD_COMMON)
 
+test_fuzz_integer_SOURCES = $(FUZZ_SUITE) test/fuzz/integer.cpp
+test_fuzz_integer_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
+test_fuzz_integer_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
+test_fuzz_integer_LDFLAGS = $(RELDFLAGS) $(AM_LDFLAGS) $(LIBTOOL_APP_LDFLAGS)
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/Makefile.test.include]: applied

File: src/Makefile.test.include [hunk 4]
```diff
@@ -237,6 +238,12 @@ test_test_dash_LDADD += $(LIBBITCOIN_ZMQ) $(ZMQ_LIBS)
 endif
 
 if ENABLE_FUZZ
+test_fuzz_block_SOURCES = $(FUZZ_SUITE) test/fuzz/block.cpp
+test_fuzz_block_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
+test_fuzz_block_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
+test_fuzz_block_LDFLAGS = $(RELDFLAGS) $(AM_LDFLAGS) $(LIBTOOL_APP_LDFLAGS)
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/Makefile.test.include]: applied

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**